components/suggestion/at_mention_provider/at_mention_suggestion.tsx

```diff
@@ -70,7 +79,7 @@ export default class AtMentionSuggestion extends Suggestion {
                         <span className='suggestion-list__icon suggestion-list__icon--large'>
                             <i
                                 className='icon icon-account-multiple-outline'
-                                title={title}
+                                title={title as ReactNode as string}
```

Comment:
lets just cast to string ${title}

Reply:
same https://github.com/mattermost/mattermost-webapp/pull/11196#discussion_r985102335